edx_rbac/models.py

```diff
@@ -55,15 +64,19 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
-class UserRoleAssignment(TimeStampedModel):
+class UserRoleAssignment(with_metaclass(UserRoleAssignmentCreator, TimeStampedModel)):
```

Comment:
is it intentional to include TimeStampedModel in this with_metaclass call?

Reply:
hmm good eye. i think i misread the documentation with this. specifically, what the 2nd argument of that should be. will give it another pass

Reply:
i think this is right?
```
Create a new class with base class base and metaclass metaclass. This is designed to be used in class declarations like this:

from six import with_metaclass

class Meta(type):
    pass

class Base(object):
    pass

class MyClass(with_metaclass(Meta, Base)):
    pass```